korman/nodes/node_python.py

```diff
@@ -787,8 +787,8 @@ def get_key(self, exporter, so):
             gui_dialog = bo.plasma_modifiers.gui_dialog
             if not gui_dialog.enabled:
                 self.raise_error(f"GUI Dialog modifier not enabled on '{self.object_name}'")
-            dialog_mod = exporter.mgr.find_create_object(pfGUIDialogMod, so=ref_so, bl=bo)
-            dialog_mod.procReceiver = attrib.node.get_key(exporter, so)
+            dialog_mod = exporter.mgr.find_create_object(pfGUIDialogMod, bl=bo)
+            dialog_mod.procReceiver = pynode.node.get_key(exporter, so)
```

Comment:
```suggestion
            dialog_mod = exporter.mgr.find_create_object(pfGUIDialogMod, so=ref_so, bl=bo)
            dialog_mod.procReceiver = attrib_socket.node.get_key(exporter, so)
```

Your pool object commit should prevent the `pfGUIDialogMod` from getting added to the modifiers list on the `ref_so`. Passing in `ref_so` will allow us to skip a dictionary lookup for the GUI page's location.